constraints.txt

```diff
@@ -24,6 +24,6 @@
 # ga4gh-client==0.1.0
 # ga4gh-schemas==0.1.0
 #
-git+git://github.com/ga4gh/schemas.git@master#egg=ga4gh_schemas
+git+git://github.com/dcolligan/schemas.git@david_774#egg=ga4gh_schemas
```

Comment:
Yeah, it seems natural that when a server PR is dependent on a schema change, the merger can update the dependencies as the merges come down.